.github/workflows/validations.yaml

```diff
@@ -23,6 +23,14 @@ jobs:
     - name: Upgrade Python packaging tools
       run: pip install --disable-pip-version-check --upgrade pip setuptools wheel
 
+    - name: Install codespell
+      run: python3 -m pip install codespell
+
+    - name: Run codespell for typos
+      run: |
+        codespell --skip="*.txt,*.json,*/.tox/*,*.sql,./site/*,*/metrics.yaml,*/CHANGELOG.md,*/.git/*,*/*.egg-info/*,*/build/*,*/vendor/*,*/tests/*" \
+          --ignore-words=.github/workflows/words.txt --ignore-regex="PullRequest|PostgresSQL"
```

Comment:
Let's https://github.com/codespell-project/codespell/#using-a-config-file with everything stored at the root in `/codespell`